File: sample/src/main/java/me/xiaopan/assemblyadaptersample/bean/UserGroup.java
package me.xiaopan.assemblyadaptersample.bean;

import java.util.List;

import me.xiaopan.assemblyexpandableadapter.AssemblyGroup;

public class UserGroup implements AssemblyGroup {
    public String title;
    public List<User> userList;

    @Override
    public int getChildCount() {
        return userList != null ? userList.size() : 0;
    }

    @Override
    public Object getChild(int childPosition) {
        return userList != null && childPosition < userList.size() ? userList.get(childPosition) : null;
    }
}

File: sample-old/src/main/java/com/github/panpf/assemblyadapter/sample/old/bean/UserGroup.kt
package com.github.panpf.assemblyadapter.sample.old.bean

import me.panpf.adapter.expandable.AssemblyGroup

class UserGroup : AssemblyGroup {
    var title: String? = null
    var userList: ArrayList<User>? = null

    override fun getChildCount(): Int {
        return if (userList != null) userList!!.size else 0
    }

    override fun getChild(childPosition: Int): Any? {
        return if (userList != null && childPosition < userList!!.size) userList!![childPosition] else null
    }
}